### PR TITLE
Conditional formatting for attribute table cells

### PR DESCRIPTION
**Pull request by yjacolin**
* Add new icon
* Add new subsection in editing_geometry_attributes chapter, Attributes
  table section

Need screenshot, #631

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -928,6 +928,8 @@
    :width: 1.5em
 .. |removeLayer| image:: /static/common/mActionRemoveLayer.png
    :width: 1.5em
+.. |rendererCategorizedSymbol| image:: /static/common/rendererCategorizedSymbol.png
```
Comment (DelazJ) [inline source/conf.py]: adding it to substitution list in guidelines?

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -1005,18 +1005,18 @@ filter:
 * Field Filter;
 * Advanced filter (Expression);
 
-The first four are self explanatory, the two last are expression filters. Field 
-Filter allow user to choose a column name in the list and add a simple form to 
-the right of the dropdown list to filter with a *like* expression parameter. 
+The first four are self explanatory, the two last are expression filters. Field
+Filter allow user to choose a column name in the list and add a simple form to
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: an old typo: allow**s**

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 3]
```diff
@@ -1005,18 +1005,18 @@ filter:
...
+the right of the dropdown list to filter with a *like* expression parameter.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: another: drop-down

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 4]
```diff
@@ -1071,6 +1071,25 @@ values, or a range of values that are allowed, to be added to a specific vector
...
 
+
+.. index:: conditional_formatting
+
+Conditional formatting of Table Cells
+......................................
+
+You can enable the conditional formatting panel clicking on
+|rendererCategorizedSymbol| at the right of the main toolbar at the top of the
+window.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: "at the top right of the main window" _instead ?_

Would need to add the condition that the panel opens only in table view mode (not form one)

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 5]
```diff
@@ -1071,6 +1071,25 @@ values, or a range of values that are allowed, to be added to a specific vector
...
+
+The new panel allows user to add new rules for conditional formatting of field
+or full row in regard of the expression on field. Adding new rule open a form
+to define a name to the rule, a condition from expression window, a preset
+formatting and some parameters to improve, change or setup the formatting
+(background  and text colors, use of icon, bold, italic underline, or strikeout
+text field and finally choose the font).
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: would a bulleted list be more readable?

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 6]
```diff
@@ -1071,6 +1071,25 @@ values, or a range of values that are allowed, to be added to a specific vector
...
+
+.. TODO: Screenshot!
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]: +1

**Comment by DelazJ** [comment]
few comments inline
Otherwise, all is there. LGTM

**Comment by SrNetoChan** [comment]
Se 70cc7be it adds a screenshot for this.

**Pull request merged**